.ci/README.md

```diff
@@ -1,63 +1,87 @@
 Concourse CI tools for MagicModules and Google Providers
 ===
 
-These tools manage the downstream repositories of [magic-modules](https://github.com/GoogleCloudPlatform/magic-modules).
+These tools manage the downstream repositories of [magic-modules](https://github.com/GoogleCloudPlatform/magic-modules), and are collectively referred to as "The Magician".
 
-# Jobs
-The concourse pipeline defined here runs through four stages when a Github pull request is opened against `magic-modules`.
-## `mm-generate`
-The first job's goal is to generate the downstream repositories of MagicModules.  This is done almost entirely by the `generate` task.
+# CI For Downstream Developers
+If you're interested in developing the repositories that manage MagicModules, here are the things you'll want to know.
```

Comment:
Should this be "the repositories that MagicModules manages?"

Reply:
Yep!